tests/performance/scripts/runperf.sh

```diff
@@ -32,7 +32,7 @@ cat >config.json<<END
 {
         "image_url": ["nginx", "hello-world"],
         "k8smasterforkubeedge": "$K8SMasterForKubeEdge",
-        "node_num": 10,
+        "node_num": 500,
```

Comment:
can we keep this as 10 itself ?

Reply:
Let's keep it 500. It is the default max nodes,  not the running nodes:)